metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_report.py

```diff
@@ -107,9 +107,9 @@ class BigQueryV2Report(
     bigquery_audit_metadata_datasets_missing: Optional[bool] = None
     lineage_failed_extraction: LossyList[str] = field(default_factory=LossyList)
     lineage_metadata_entries: TopKDict[str, int] = field(default_factory=TopKDict)
-    lineage_mem_size: Dict[str, str] = field(default_factory=TopKDict)
-    lineage_extraction_sec: Dict[str, float] = field(default_factory=TopKDict)
-    usage_extraction_sec: Dict[str, float] = field(default_factory=TopKDict)
+    lineage_mem_size: TopKDict[str, str] = field(default_factory=TopKDict)
```

Comment:
TopKDict probably won't do what you want it to for string types

Reply:
removed this